rust-src/concordium_base/src/pedersen_commitment/key.rs

```diff
@@ -17,6 +17,17 @@ pub struct CommitmentKey<C: Curve> {
     pub h: C,
 }
 
+/// A vector commitment key is a list of group elements that are used as bases
+/// to raise the values and randomness, respectively.
+#[derive(Debug, PartialEq, Eq, Clone, Serialize, SerdeBase16Serialize)]
+pub struct VecCommitmentKey<C: Curve> {
+    /// Bases to raise the values to when committing.
+    #[size_length = 4]
+    pub gs: Vec<C>,
```

Comment:
I suppose we assume this is not empty, right? Can we add this to the documentation.

Reply:
Yes. Should always be non-empty. Added this assumption.